dashboard/app.R

```diff
@@ -334,30 +336,53 @@ server <- function(input, output, session) {
         locationSubtitleText = paste0(', Location: ', aggregate ,' over all states and territories common to these forecasters*')
         output$renderLocations <- renderText(toupper(locationsIntersect))
         output$renderWarningText = renderText("")
+        showElement("truthPlot")
```

Comment:
It would be nice for the truth plot to have a legend for the as-of vs most recent data.

Reply:
That is coming in the next PR adding forecasts